docs/aws/audit/iamcompliance/rules/elb_certificates_rotation_remediation.mdx

```diff
@@ -1,6 +1,104 @@
+3. In the IAM dashboard, select "SSL/TLS certificates" from the left-hand navigation pane.
+4. Here, you can see all the SSL/TLS certificates associated with your account. Check the expiration dates of these certificates. If any of them are close to their expiration date or have already expired, it indicates that the ELB Certificates need to be rotated.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. First, you need to install and configure AWS CLI on your local machine. You can do this by following the instructions provided by AWS. Make sure you have the necessary permissions to access the resources.
+
+2. Once the AWS CLI is set up, you can list all the server certificates in IAM using the following command:
+
+   ```
+   aws iam list-server-certificates
+   ```
+
+   This command will return a JSON object that contains all the server certificates.
+
+3. To check the expiration date of each certificate, you can use the following command:
+
+   ```
+   aws iam get-server-certificate --server-certificate-name {certificate-name}
+   ```
+
+   Replace `{certificate-name}` with the name of the certificate you want to check. This command will return a JSON object that contains the details of the certificate, including the expiration date.
+
+4. You can then write a Python script to parse the JSON object and check if the certificate is expired or not. Here is a simple example:
+
+   ```python
+   import json
+   import subprocess
+   from datetime import datetime
+
+   # Run the AWS CLI command and get the output
+   output = subprocess.check_output(['aws', 'iam', 'get-server-certificate', '--server-certificate-name', 'your-certificate-name'])
+
+   # Parse the JSON output
+   data = json.loads(output)
+
+   # Get the expiration date
+   expiration_date = data['ServerCertificate']['ServerCertificateMetadata']['Expiration']
+
+   # Convert the expiration date to a datetime object
+   expiration_date = datetime.strptime(expiration_date, '%Y-%m-%dT%H:%M:%S%z')
+
+   # Check if the certificate is expired
+   if expiration_date < datetime.now():
+       print('The certificate is expired.')
+   else:
+       print('The certificate is not expired.')
+   ```
+
+   Replace `'your-certificate-name'` with the name of your certificate. This script will print whether the certificate is expired or not.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Import the necessary libraries: You will need the boto3 library, which allows Python developers to write software that makes use of services like Amazon S3, Amazon EC2, etc.
+
+```python
+import boto3
+from datetime import datetime, timedelta
+```
+
+2. Create an AWS session using boto3: You can create a session using your AWS credentials. Replace 'aws_access_key_id', 'aws_secret_access_key', and 'region_name' with your AWS credentials.
+
+```python
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target="_blank">Click here for more details</a></b>